internal/metrics/api.go

```diff
@@ -81,39 +85,51 @@ func (cl *APIClient) SendMetrics(metrics []APIMetric) error {
 	}
 	body := bytes.NewBuffer(content)
 
-	// For the moment we only support distribution metrics.
-	// Other metric types use the "series" endpoint, which takes an identical payload.
-	req, err := http.NewRequest("POST", cl.makeRoute("distribution_points"), body)
-	if err != nil {
-		return fmt.Errorf("Couldn't create send metrics request:%v", err)
+	if breaker == nil {
+		var st gobreaker.Settings
```

Comment:
Do you mind making the Settings explicitly defined, so we do not rely on the defaults?